src/test/scala/net/ruippeixotog/scalascraper/browser/BrowserSpec.scala

```diff
@@ -237,6 +237,25 @@ class BrowserSpec extends Specification with BrowserHelper with TestServer {
           textNode.innerHtml mustEqual "this is <b>some</b> text"
           textNode.outerHtml mustEqual "<span id=\"t\">this is <b>some</b> text</span>"
         }
+
+        "with correct siblings methods" in {
+          val doc = browser.parseString("""<html>
```

Comment:
Instead of inlining the HTML example here, can you modify the `html` variable above, adding a new div somewhere with the example you need? I'd rather not inline HTML in each example, purely for code organization reasons.